### ISSUE TITLE
`src/Application/Actions/ActionError` has wrong type `$description`

### ISSUE THREAD

**Issue by Dmitrev** [slimphp/Slim-Skeleton#289]
```php
// src/Application/Actions/ActionError.php

class ActionError implements JsonSerializable
{
    
    private string $description;                                     // <- string

    public function __construct(string $type, ?string $description)  // string|null ?
    {
        $this->type = $type;
        $this->description = $description;
    }

 
    public function getDescription(): string                          // string
    {
        return $this->description;
    }

    public function setDescription(?string $description = null): self // string|null ?
    {
        $this->description = $description;
        return $this;
    }
}
```

This file declares `$description` as `string`

https://github.com/slimphp/Slim-Skeleton/blob/master/src/Application/Actions/ActionError.php#L23

While all the other methods seem to accept `?string`
https://github.com/slimphp/Slim-Skeleton/blob/master/src/Application/Actions/ActionError.php#L25
https://github.com/slimphp/Slim-Skeleton/blob/master/src/Application/Actions/ActionError.php#L47

However passing a null to these methods will result in a `TypeError`

Is this a typo or intentional? In my own project I just converted everything to `string`. Happy to make a PR for that if you agree.

**Comment by nbayramberdiyev** [comment]
Hey, thanks for raising this issue. Feel free to open a PR.

**Comment by Dmitrev** [comment]
Fixed in #290
